ansible-playbooks/ros_disablement.yml

```diff
@@ -0,0 +1,41 @@
+# This playbook will take care of all steps required to remove
+# Insights resource-optimization (pcp packages)
+# Optionally with the tag "clean-up", it will remove the ROS config file as well
+
+- hosts: all
+  become: yes
+  tasks:
+    - name: Ensure PCP service is stopped
+      systemd:
+        state: stopped
+        enabled: no
+        name: "{{ item }}"
+      with_items:
+        - 'pmcd'
+        - 'pmlogger'
+      tags:
+        - always
+    - name: Remove ROS pmlogger service instance
+      lineinfile:
+        dest: /etc/pcp/pmlogger/control.d/local
+        regexp: ".*config.ros.*"
+        state: absent
+      tags:
+        - always
+    - name: Remove ROS configuration file
+      file:
+        state: absent
+        path: /var/lib/pcp/config/pmlogger/config.ros
+      when: "'clean-up' in ansible_run_tags"
```

Comment:
@saltgen, @apuntamb -
one query - As we have introduced new state `waiting for data` where it checks for a ros config file presence.  When end-user wants to disable ROS service, does that make sense **not to** include this task as part of default behaviour? 
OR 
Is it correct to show `waiting for data` on next run as this config file will be present? In this scenario, I think it will make flag `is_ros=true`. Please confirm.

Reply:
@kgaikwad Not too sure about this one, will check and revert shortly

Reply:
@saltgen - as per offline discussion, this step should be part of default tasks while disabling ROS from a system. With additional tag, this playbook should remove pcp package.

Reply:
@kgaikwad Changes pushed :+1: 

Suggesting clone and validate even though if we do this MR right respective user will no longer a part of ROS :wink: